src/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php

```diff
@@ -358,7 +358,13 @@ private function addServiceReturn($id, $definition)
      */
     private function addServiceInstance($id, $definition)
     {
-        $class = $this->dumpValue($definition->getClass());
+        $class = $definition->getClass();
+
+        if ('\\' === substr($class, 0, 1)) {
```

Comment:
I think we should treat more than one leading backslash as an error and not accept it (it might be a hint for another mistake that the user made).